AABB_tree/include/CGAL/AABB_tree.h

```diff
@@ -632,6 +656,30 @@ namespace CGAL {
 #endif
   {}
 
+  template <typename Tr>
+  typename AABB_tree<Tr>::Self& AABB_tree<Tr>::operator=(Self&& tree) noexcept
+  {
+    m_traits = std::move(tree.m_traits);
+    m_primitives = std::move(tree.m_primitives);
+    m_nodes = std::move(tree.m_nodes);
+    m_p_search_tree = std::move(tree.m_p_search_tree);
+    m_use_default_search_tree = std::exchange(tree.m_use_default_search_tree, true);
+#ifdef CGAL_HAS_THREADS
+    m_atomic_need_build = tree.m_atomic_need_build.load(std::memory_order_relaxed);
```

Comment:
I'm not sure that what I did is the best choice.

Reply:
That is correct as far as I can see.